Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM jupyter/datascience-notebook:lab-2.2.9
+
+USER ${NB_USER}
+
+COPY --chown=${NB_USER}:users ./plutoserver ./plutoserver
+COPY --chown=${NB_USER}:users ./environment.yml ./environment.yml
+COPY --chown=${NB_USER}:users ./setup.py ./setup.py
+COPY --chown=${NB_USER}:users ./runpluto.sh ./runpluto.sh
+
+RUN julia -e "import Pkg; Pkg.add([\"PlutoUI\", \"Pluto\", \"DataFrames\", \"CSV\", \"Plots\"]); Pkg.precompile()"
+
+RUN pip install --no-cache-dir .
```

Comment:
To get the Pluto button to appear, I had to add the following lines:

```docker
RUN jupyter labextension install @jupyterlab/server-proxy
RUN jupyter lab build
```

Unless I'm missing something, I think these are necessary, or how do you start Pluto?

Reply:
Oh I was wondering how that works, thanks!

It seems that the pip install is enough to get pluto on the `/pluto` endpoint, but not the lab extension.
I'll add that, thank you!!

Reply:
Aha, I didn't know about the /pluto endpoint, that's good to know!